docs/migrations/keyring.md

```diff
@@ -0,0 +1,32 @@
+# Keyring Migrate Quick Start
+
+`keyring` is mechanism of managing private/public keypair in cosmos-sdk. 
+
+The more detailed information about keyring you can find here [here](../run-node/keyring.md)
+
+
+## Backends
+
+The `keyring` supports the following backends:
+
+* `os` uses the operating system's default credentials store.
+* `file` uses encrypted file-based keystore within the app's configuration directory. This keyring will request a password each time it is accessed, which may occur multiple times in a single command resulting in repeated password prompts.
+* `kwallet` uses KDE Wallet Manager as a credentials management application.
+* `pass` uses the pass command line utility to store and retrieve keys.
+* `test` stores keys insecurely to disk. It does not prompt for a password to be unlocked and it should be use only for testing purposes.
```

Comment:
I don't think we need to discuss about this here, it's all specified in ../run-node/keyring.md

Reply:
Ok